modules/ocf/manifests/extrapackages.pp

```diff
@@ -219,6 +216,30 @@
     ]:;
   }
 
+  if $::lstdistcodename == 'stretch' {
+    package {
+      [
+        # php-mcrypt is deprecated since PHP 7.1 in favor of using openssl
+        # instead and buster has PHP 7.3:
+        # http://php.net/manual/en/migration71.deprecated.php
+        'php-mcrypt',
+
+        # pkpgcounter comes from pykota, which is unmaintained upstream, so we
+        # need to find a replacement or forward port it for buster. We had this
+        # same problem with pykota from wheezy to jessie but with pykota as a
+        # broader package. Looks like pkpgcounter is finally joining pykota and
+        # we should move off it somehow and find some other way to count pages.
+        'pkpgcounter',
```

Comment:
Where is this used by us? A Sourcegraph search gave me nothing.

Reply:
Hmmm, yeah I can't find anything either, but I thought it was pretty integral to the printing pipeline. Then again, I'm not sure why it's in `ocf::extrapackages` then and not in just `ocf_printhost`...

I'll try and do some more digging

Reply:
Ok yeah I think we're just counting the pages using postscript now with [ocf_printhost/files/enforcer-pc](https://github.com/ocf/puppet/blob/master/modules/ocf_printhost/files/enforcer-pc), so that's great. We can totally remove this then, I don't believe it's used anywhere. Thanks for pointing that out!